src/hooks/screen/useChatScreenHook.ts

```diff
@@ -247,24 +258,17 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
               refresh('channelList');
             }
           } else {
-            queue.addPriorityJob({
-              operationLabel: DatabaseOperationLabel.ChatScreen_UpdateChannelDescription,
-              id: `${selectedChannel?.id}-${randomId}`,
-              priority: QueueJobPriority.HIGH,
-              task: async () => {
-                const channelList: ChannelList | null = await ChannelList.getSchemaById(
-                  localDb,
-                  selectedChannel?.id
-                );
-                if (channelList) {
-                  channelList.description = message;
-                  channelList.lastUpdatedBy = userId;
-                  channelList.lastUpdatedAt = new Date().toISOString();
-                  await channelList.save(localDb);
-                  refresh('channelList');
-                }
-              }
-            });
+            const channelList: ChannelList | null = await ChannelList.getSchemaById(
+              localDb,
+              selectedChannel?.id
+            );
+            if (channelList) {
+              channelList.description = message;
+              channelList.lastUpdatedBy = userId;
+              channelList.lastUpdatedAt = new Date().toISOString();
+              await channelList.save(localDb);
+              refresh('channelList');
+            }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The usage of `useDatabaseQueueHook` has been replaced with direct database operations. This simplifies the code and makes it easier to understand. However, it's important to ensure that the removal of `useDatabaseQueueHook` does not affect the application's performance or functionality.
```diff
-            queue.addPriorityJob({
-              operationLabel: DatabaseOperationLabel.ChatScreen_UpdateChannelDescription,
-              id: `${selectedChannel?.id}-${randomId}`,
-              priority: QueueJobPriority.HIGH,
-              task: async () => {
-                const channelList: ChannelList | null = await ChannelList.getSchemaById(
-                  localDb,
-                  selectedChannel?.id
-                );
-                if (channelList) {
-                  channelList.description = message;
-                  channelList.lastUpdatedBy = userId;
-                  channelList.lastUpdatedAt = new Date().toISOString();
-                  await channelList.save(localDb);
-                  refresh('channelList');
-                }
-              }
-            });
+            const channelList: ChannelList | null = await ChannelList.getSchemaById(
+              localDb,
+              selectedChannel?.id
+            );
+            if (channelList) {
+              channelList.description = message;
+              channelList.lastUpdatedBy = userId;
+              channelList.lastUpdatedAt = new Date().toISOString();
+              await channelList.save(localDb);
+              refresh('channelList');
+            }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->